x-pack/plugins/apm/public/components/app/TransactionDetails/Distribution/index.tsx

```diff
@@ -87,7 +87,7 @@ export class Distribution extends Component<Props> {
       <div>
         <EuiTitle size="s">
           <h5>
-            Response time distribution{' '}
+            Transactions duration distribution{' '}
```

Comment:
Maybe I've just gotten used to "Response time distribution" but I'm not sure I'd understand what "Transactions duration distribution" was without the former.